fixes and improvements

cahein opened the pull request:
Greetings,
using the spree_product_zoom extension, we found several problems.
Most obvious is the overwriting of the 'zoom.gif' below the main image (through showVariantImages in product.js.coffee).

Also, the behavior in different scenarios of product/variants was not like we need it.

Added a features test to document the current functioning of the extension. Unfortunately, the product images are not automatically uploaded yet and need to be copied into the dummy manually.

cahein committed:
fix 'zoom.gif'

cahein committed:
handle variant images. use first variant

cahein committed:
zoom.gif overwrite problem fix

cahein committed:
zoom only currently selected variant images

cahein committed:
variant gallery fix

cahein committed:
add features test

cahein committed:
minor test change

cahein committed:
always include master variant images in gallery

cahein committed:
add expect fancybox function

futhr commented on spree_product_zoom.gemspec:
```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.requirements << 'none'
 
-  s.add_dependency 'spree_core', '~> 2.4.0.beta'
+  s.add_dependency 'spree_core', '~> 2.2'
```
You need to remove this line it already set to 2.4.0, then we can create a 2-4-stable branch from this.

cahein committed:
spree 2.4

cahein committed:
update to spree 2.4

cahein committed:
move assets

cahein committed:
test setup

cahein commented:
moved the style and javascript assets and changed the install_generator accordingly
fixed the spec_helper to copy required test assets to the dummy app

JDutil commented on lib/spree_product_zoom/factories.rb:
```diff
@@ -0,0 +1,45 @@
+FactoryGirl.define do
```
FYI in your spec helper your can require spree's factories:
https://github.com/spree-contrib/spree_price_books/blob/master/spec/spec_helper.rb#L28

cahein committed:
update info

cahein committed:
update gemspec and tests

cahein committed:
minor change

cahein committed:
version requirements
